packages/monitoring-system/metrics-dashboard/src/index.html

```diff
@@ -1,95 +1,16 @@
 <html>
```

Comment:
This diff is weird because this `index.html` is a completely new HTML file that contains the components for the auth workflow. Ideally I'd like to name this `auth.html` but apparently you cannot rename the homepage in Firebase Hosting